ccm_web/nmon_config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
what is the purpose of this file?

Reply:
This configures `nodemon`, which is how changes to the backend (`app.ts`) re-load. See the `dev` script in `package.json`.